JeMPI_Apps/JeMPI_UI/src/components/import/DropZone.tsx

```diff
@@ -189,23 +191,84 @@ const DropZone: FC = () => {
           <Typography fontSize="16px">
             <a>Click to upload</a> or drag and drop
           </Typography>
-          <Typography color="#00000099" fontSize="14px">
+          <Typography color="#00000099" fontSize="1rem">
             CSV (max. 128MB)
```

Comment:
Should we not reference `MAX_UPLOAD_FILE_SIZE_IN_BYTES ` here

Reply:
yes, this issue will be addressed.

Reply:
yes nice catch, this issue will be addressed.

Reply:
the issue is addressed in this commit, thank you again .https://github.com/jembi/JeMPI/pull/126/commits/54a1cc053c59b4cbcdceed371f5a3b7cd8944729